fluidBuild.config.cjs

```diff
@@ -159,9 +167,17 @@ module.exports = {
 				// Can be removed once the policy handler is updated to support tsc-multi as equivalent to tsc.
```

Comment:
Nit: might be worth noting the work item number tracking this cleanup for reference.